src/sentry/api/urls.py

```diff
@@ -1417,6 +1420,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         OrganizationSamplingProjectRatesEndpoint.as_view(),
         name="sentry-api-0-organization-sampling-project-rates",
     ),
+    re_path(
+        r"^(?P<organization_id_or_slug>[^\/]+)/sampling/project-span-counts/$",
```

Comment:
This endpoint returns the span count per _root_ project, which distinguishes it from regular stats endpoints. Should we choose a URL that makes this clear? This might make the URL too long, so I'm not sure we should actually do this. 

Options could be:
- `project-root-counts` (we can equally add other counts there if we need to)
- `span-root-counts`
- `root-counts`

Reply:
Yeah that's a good point. None of them really feel intuitive to me, but I suppose `project-root-counts` is probably the best, as it even takes out the specificity of spans in case we ever adapt what type of data sampling acts on.